cmd/preview/cmdpreview.go

```diff
@@ -148,7 +142,7 @@ func (r *PreviewRunner) RunE(cmd *cobra.Command, args []string) error {
 
 		// Run the applier. It will return a channel where we can receive updates
 		// to keep track of progress and any issues.
-		ch = a.Run(ctx, inv, objs, apply.ApplierOptions{
+		ch = a.Run(ctx, objs, apply.ApplierOptions{
```

Comment:
So how should we think about the information about inventory (being either the existing inventory information or a resource among the other resources with the special annotation)?

Are we providing the KRM resource that will be applied (Though in practice it will be a Replace rather than Apply/Patch) in the cluster, with the framework just adding additional data (i.e. the inventory) to the resource prior to updating the live state?

Or is the inventory resource resource just a place where we specify the key information needed to create, find, update, or delete the resource containing the inventory, but not providing the inventory resource itself?

I've always thought of it as the latter, where the information passed into the `Run` function (either as a KRM resource or as the InventoryInfo) just contains the key pieces of information needed (name, namespace and id), but it was up to the InventoryClient to manage the actual inventory resource in the cluster. I've been uncomfortable with the implementation, exactly because it uses the `InventoryInfo` interface to wrap a KRM resource.